components/icons/catalogue.md

```diff
@@ -38,82 +38,81 @@ Size: `18px` x `18`px
 
```

Comment:
Weird... why is this file here?  Do the translation PRs use out-of-date branches instead of branching of master each time?

Reply:
That _is_ weird... I'm not sure.